src/microbe_stage/Endosome.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Godot;
+using Newtonsoft.Json;
+
+/// <summary>
+///   This does nothing (for now) and only exist so saving could work.
+/// </summary>
+[JSONAlwaysDynamicType]
+[SceneLoadedClass("res://src/microbe_stage/Endosome.tscn", UsesEarlyResolve = false)]
+public class Endosome : Spatial, IEntity
+{
+    [JsonProperty]
+    private Color tint;
+
+    [JsonProperty]
+    private int renderPriority;
+
+    [JsonIgnore]
+    public MeshInstance? Mesh { get; private set; }
+
+    [JsonIgnore]
+    public Color Tint
+    {
+        get => tint;
+        set
+        {
+            tint = value;
+            ApplyTint();
+        }
+    }
+
+    [JsonIgnore]
+    public int RenderPriority
+    {
+        get => renderPriority;
+        set
+        {
+            renderPriority = value;
+            ApplyRenderPriority();
+        }
+    }
+
+    [JsonIgnore]
+    public Spatial EntityNode => this;
+
+    [JsonIgnore]
+    public AliveMarker AliveMarker { get; } = new();
+
+    public override void _Ready()
+    {
+        Mesh = GetNode<MeshInstance>("Vacuole");
+
+        // This has to be done here because setting this in Godot editor truncates
+        // the number to only 3 decimal places.
+        var material = Mesh?.MaterialOverride as ShaderMaterial;
+        material?.SetShaderParam("jiggleAmount", 0.0001f);
```

Comment:
I think at least printing an error would be good if material happens to be null so that the call here can't silently fail to update the thing, or is it supposed to be allowed to fail silently in some cases?

Reply:
No it shouldn't fail. Added print.